emsdk_manifest.json

```diff
@@ -494,13 +490,9 @@
     "append_bitness": false,
     "windows_url": "https://github.com/emscripten-core/emscripten/archive/%tag%.zip",
     "unix_url": "https://github.com/emscripten-core/emscripten/archive/%tag%.tar.gz",
-    "activated_cfg": "EMSCRIPTEN_ROOT='%installation_dir%';EMSCRIPTEN_NATIVE_OPTIMIZER='%installation_dir%%generator_prefix%_64bit_optimizer/%cmake_build_type_on_win%optimizer%.exe%'",
+    "activated_cfg": "EMSCRIPTEN_ROOT='%installation_dir%'",
     "activated_path": "%installation_dir%",
-    "activated_env": "EMSCRIPTEN=%installation_dir%;EMSCRIPTEN_NATIVE_OPTIMIZER=%installation_dir%%generator_prefix%_64bit_optimizer/%cmake_build_type_on_win%optimizer%.exe%",
-    "cmake_build_type": "Release",
-    "custom_install_script": "emscripten_post_install",
```

Comment:
This used to run `npm install` at the end for us. Was that redundant?

Reply:
Ah yes.  I changed to `emscripten_npm_install` which runs just that part.